packages/itwin/tree-widget/src/components/header/TreeHeader.tsx

```diff
@@ -3,32 +3,19 @@
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 import * as React from "react";
-import { IconButton } from "../IconButton";
 import { SearchBar } from "../search-bar/SearchBar";
 import type { SearchOptions } from "../TreeFilteringState";
 import { TreeWidget } from "../../TreeWidget";
 import "./TreeHeader.scss";
 
 export interface TreeHeaderComponentProps {
-  showAll?: () => void;
-  hideAll?: () => void;
-  invert?: () => void;
-  toggle2D?: () => void;
-  toggle2DIcon?: string;
-  toggle3D?: () => void;
-  toggle3DIcon?: string;
   searchOptions: SearchOptions;
+  treeHeaderButtons: React.ReactNode[];
```

Comment:
I thinks after changes to `SearchBar` `TreeHeaderComponent` can also take buttons as `children` instead of taking them through props

Reply:
fixed